server_admin/topics/user-federation/ldap.adoc

```diff
@@ -137,6 +137,14 @@ FullName Mapper::
   This allows you to specify that the full name of the user, which is saved in some LDAP attribute (usually `cn` ) will be mapped to `firstName` and `lastname` attributes in the {project_name} database.
   Having `cn` to contain full name of user is a common case for some LDAP deployments.
 
+NOTE: When registering new users in {project_name} and `Sync Registrations` switch is ON for the LDAP provider, the fullName mapper
+      allows the possibility of fallback to the username. This is especially useful in case of the Microsoft Active Directory. The common
+      setup for the MSAD is to configure `cn` LDAP attribute as fullName and at the same type, the `cn` is usually used as `RDN LDAP Attribute`
```

Comment:
Should the 'cn' is usually.. be a separate sentence?

Reply:
I've figured I did a typo in the word "type", which should be "time" . This means the sentence had a different sense than it should have :)

So I've just changed "type" to "time" and the sentence is now like: The common setup for the MSAD is to configure `cn` LDAP attribute as fullName and at the same time, the `cn` is usually ...

Is it better this way?